pages/[...slug].vue

```diff
@@ -18,6 +18,39 @@
   <EpPageMeta :doc="doc" />
   <NuxtLayout>
     <div class="px-4 print:px-8 mt-10 mb-96 print:mb-0">
+      <div>
+        <nav>
+          <span v-if="breadcrumbs.length != 1">
+            <NuxtLink :to="'/'">
+              <Icon name="heroicons:home-20-solid" size="2em" />
+            </NuxtLink>
+            {{ '>' }}
+          </span>
+
+          <span
+            v-for="x in breadcrumbs.slice(1, breadcrumbs.length - 1)"
+            :key="x._path"
+            class="flex-row m-1"
+          >
+            <NuxtLink
+              class="underline text-secondary-600 text-lg"
+              :to="x._path"
+            >
+              {{ x.title }}
+            </NuxtLink>
+            {{ '>' }}
+          </span>
+
+          <span
+            v-for="x in breadcrumbs.slice(breadcrumbs.length - 1)"
```

Comment:
You don't need iteration for last item of array. 

You also can delete slice from previous iteration and do something special if index is equal to index of the last element.

```vue
         <span
            v-for="(crumb, index) in breadcrumbs"
            :key="crumb._path"
            class="flex-row m-1"
          >
            <template v-if="index < breadcrumbs.length - 1">
              <NuxtLink
                class="underline text-secondary-600 text-lg"
                :to="crumb._path"
              >
              {{ crumb.title }}
              </NuxtLink>
              >
            </template>
            <span v-else>{{ crumb.title }}</span>
          </span>
```